src/reflect/scala/reflect/internal/util/Statistics.scala

```diff
@@ -177,22 +179,37 @@ quant)
   }
 
   class Timer(val prefix: String, val phases: Seq[String]) extends Quantity {
-    var nanos: Long = 0
-    var timings = 0
-    def start() = {
-      (nanos, System.nanoTime())
+    private val threadNanos = new ThreadLocal[LongRef] {
+      override def initialValue() = {
+        totalThreads.incrementAndGet()
+        new LongRef(0)
+      }
+    }
+    def nanos = totalNanos.get
```

Comment:
Nit: `def` should follow `val` to avoid future NPEs when hacking around this area.